src/Microsoft.IdentityModel.Logging.Abstractions/IIdentityLogger.cs

```diff
@@ -0,0 +1,49 @@
+﻿//------------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.IdentityModel.Logging.Abstractions
+{
+    /// <summary>
+    /// Interface for Logging.
+    /// </summary>
+    public interface IIdentityLogger
+    {
+        /// <summary>
+        /// Checks to see if logging is enabled at given <paramref name="eventLevel"/>.
+        /// </summary>
+        /// <param name="eventLevel">Log level of an Event.</param>
+        bool IsEnabled(EventLevel eventLevel);
```

Comment:
App developer typically provide a minimal log level. I propose we change to

```csharp
EventLevel LogVerbosity { get; }
```

Reply:
Would it be possible to keep the current method signature? ASP.Net Core's ILogger.IsEnabled() method was taken as reference when creating this as we write to their ILogger. 
Link - https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.logger-1.microsoft-extensions-logging-ilogger-isenabled?view=dotnet-plat-ext-6.0#microsoft-extensions-logging-logger-1-microsoft-extensions-logging-ilogger-isenabled(microsoft-extensions-logging-loglevel)

Reply:
This property can be added to a class that is implementing the IIdentityLogger interface and IsEnabled(currentEventLevel) could check currentEventLevel >= LogVerbosity.

ASP.Net Core's ILogger.IsEnabled(..) was used a reference when creating this method to keep the IIdentityLogger interface close to what ASP.Net Core has without explicitly taking a dependency on it.

Link - https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.logger-1.microsoft-extensions-logging-ilogger-isenabled?view=dotnet-plat-ext-6.0#microsoft-extensions-logging-logger-1-microsoft-extensions-logging-ilogger-isenabled(microsoft-extensions-logging-loglevel)